src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,51 @@
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+li {
+  list-style: none;
+}
+
+img {
+  vertical-align: top;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
```

Comment:
If you do not use any tags in the code, you do not need to set styles for them. 
Do you really need to reset the styles for h1, h3, h4, h5, h6?
Because I don't see them in the code right now.